_dev/back/back.scss

```diff
@@ -301,6 +304,9 @@
 .listing-row div img {
     width: 35px;
 }
+.listing-row .material-icons:hover {
+    color:#25b9d7;
```

Comment:
```suggestion
    color: #25b9d7;
```

Reply:
I copied this raw color from `<i class="material-icons">mode_edit</i>`'s hover inspector in the last column.
Maybe we need replace blockreassurance's `"material-design-icons": "^3.0.1"` by  `"material-design-icons-iconfont": "^6.1.0"` like in admin-dev default theme already did since 1.7.8  in another PR.